client/src/pages/Login/Login.tsx

```diff
@@ -36,29 +37,29 @@ export default function Login(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box
-          display="flex"
-          justifyContent="space-between"
-          alignItems="flex-start"
-          flexDirection="column"
-          className={classes.authWrapper}
-        >
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
+    <>
+      <Navbar></Navbar>
+      <Grid container component="main" className={classes.root}>
+        <Grid item xs={12} component={Paper} square className={classes.paper}>
+          <Box
+            display="flex"
+            justifyContent="space-between"
+            alignItems="flex-start"
+            flexDirection="column"
+            className={classes.authWrapper}
+          >
+            <Box width="100%" maxWidth={600} p={3} alignSelf="center" className={classes.loginContainer}>
+              <Grid container>
+                <Grid item xs>
+                  <h2 className={classes.welcome}>Sign in!</h2>
```

Comment:
We can still notice the usage of plain HTML elements, please use MUI components where applicable.

For eg: `Box` components instead of `div`s and `Typography` components instead of HTML header tags